### ISSUE TITLE
[Feature] ICommandGenerator should support canExecute relay command argument

### ISSUE THREAD

**Issue by DavidZidar** [CommunityToolkit/dotnet#10]
## Describe the problem this feature would solve

Currently there seems to be no way of using this source generator for relay commands that need to use the `canExecute` argument of `RelayCommand`/`AsyncRelayCommand`. This means that many commands will still have to be implemented the old way and now there are two ways of doing commands in a given project.

## Describe the solution

I imagine a potential solution would be adding an optional parameter to `ICommandAttribute`, example:

```csharp
private Task<bool> CanGreetUserAsync()
{
}

[ICommand(CanExecute = nameof(CanGreetUserAsync))]
private Task GreetUserAsync()
{
}
```

## Describe alternatives you've considered

I guess it could also be inferred based on name alone, but that might not work well if code is written in other languages.

**Comment by Sergio0694** [comment]
Forgot to link this issue, the feature is now available and was added in #50 😊

**Comment by DavidZidar** [comment]
@Sergio0694 Nice! Thank you for the update! 😄
